packages/gui/src/components/nfts/NFTCard.tsx

```diff
@@ -59,7 +61,13 @@ export default function NFTCard(props: NFTCardProps) {
         ) : (
           <>
             <CardActionArea onClick={handleClick}>
-              <NFTPreview nft={nft} fit="cover" />
+              <NFTPreview
+                nft={nft}
+                fit="cover"
+                isPreview
+                metadata={metadata || error}
```

Comment:
If `error` needs to be passed as a prop, it should be passed as its own prop. Otherwise it makes it complicates the logic for handling the metadata or error in NFTPreview.

Reply:
Done.